analyzer/src/main/kotlin/managers/PhpComposer.kt

```diff
@@ -312,18 +312,16 @@ class PhpComposer(
     private fun parseDeclaredLicenses(packageInfo: JsonNode) =
         packageInfo["license"]?.mapNotNull { it?.textValue() }?.toSortedSet() ?: sortedSetOf<String>()
 
-    private fun parseVcsInfo(packageInfo: JsonNode): VcsInfo {
-        return packageInfo["source"]?.let {
+    private fun parseVcsInfo(packageInfo: JsonNode) =
```

Comment:
prefer explicit return type here

Reply:
For me, the return type is obvious here as the "else" branch has `VcsInfo.EMPTY`.